packages/errors/src/errors.ts

```diff
@@ -351,6 +351,30 @@ export const AllCypressErrors = {
 
         https://on.cypress.io/run-group-name-not-unique`
   },
+  CLOUD_AUTO_CANCEL_NOT_AVAILABLE_IN_PLAN: (arg1: {link: string}) => {
+    return errTemplate`\
+      ${fmt.highlightSecondary(`Auto Cancellation`)} is not included under your current billing plan.
+      To enable this service, please visit your billing and upgrade to another plan with Auto Cancellation.
+      ${fmt.off(arg1.link)}`
```

Comment:
Why isn't this link in the error message? Is this expected to change over time?

Reply:
This is the link that is being passed https://github.com/cypress-io/cypress/blob/a2b58f0e1875efa30c410aad2fb8939407a65af5/packages/server/lib/modes/record.js#L431 - which is dynamically build using the orgId